CHANGELOG.md

```diff
@@ -24,27 +70,6 @@
 
 - Refs [\#28067](https://projects.theforeman.org/issues/28067): Ensure dynflow worker config exists before service [\#791](https://github.com/theforeman/puppet-foreman/pull/791) ([ehelms](https://github.com/ehelms))
 
-## [13.1.0](https://github.com/theforeman/puppet-foreman/tree/13.1.0) (2019-11-25)
-
-[Full Changelog](https://github.com/theforeman/puppet-foreman/compare/13.0.1...13.1.0)
-
-**Implemented enhancements:**
-
-- Add Foreman AzureRM cli option [\#772](https://github.com/theforeman/puppet-foreman/pull/772) ([apuntamb](https://github.com/apuntamb))
-- Initial AzureRM support [\#767](https://github.com/theforeman/puppet-foreman/pull/767) ([apuntamb](https://github.com/apuntamb))
-
-**Fixed bugs:**
-
-- Fixes [\#28200](https://projects.theforeman.org/issues/28200) - Change cockpit port from 9999 to 19090 [\#768](https://github.com/theforeman/puppet-foreman/pull/768) ([adamruzicka](https://github.com/adamruzicka))
-
-## [13.0.1](https://github.com/theforeman/puppet-foreman/tree/13.0.1) (2019-10-31)
-
-[Full Changelog](https://github.com/theforeman/puppet-foreman/compare/13.0.0...13.0.1)
-
-**Fixed bugs:**
-
-- Fixes [\#28146](https://projects.theforeman.org/issues/28146) - Drop double leading slash from cockpit url [\#764](https://github.com/theforeman/puppet-foreman/pull/764) ([adamruzicka](https://github.com/adamruzicka))
-
```

Comment:
This should remain

Reply:
I copied the entire changelog to HISTORY.md so this should get preserved now